package.json

```diff
@@ -133,13 +134,15 @@
     "react-redux": "^7.2.1",
     "react-router-dom": "^5.2.0",
     "react-smartbanner": "^5.1.4",
+    "react-sortablejs": "^6.0.0",
     "react-toastify": "^6.0.9",
     "react-tooltip": "^4.2.9",
     "redux": "^4.0.5",
     "redux-debounced": "^0.5.0",
     "redux-mock-store": "^1.5.4",
     "redux-thunk": "^2.3.0",
-    "reselect": "^4.0.0"
+    "reselect": "^4.0.0",
+    "sortablejs": "^1.13.0"
```

Comment:
Does `react-sortablejs` needs `sortablejs` library along, thought it is already part of `react-sortablejs` itself?

Reply:
Yeah both need to install separately https://github.com/SortableJS/react-sortablejs